lms/djangoapps/teams/views.py

```diff
@@ -1094,7 +1095,7 @@ def get(self, request):
         if requested_course_key is not None:
             course_keys = [requested_course_key]
         elif accessible_course_ids is not None:
-            course_keys = [CourseKey.from_string(course_string) for course_string in accessible_course_ids]
+            course_keys = [course_key_from_string_or_404(course_string) for course_string in accessible_course_ids]
```

Comment:
we can assume this are not faulty course_strings. No need to use the helper here.